scripts/setup_sgtm_tasks_project.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+
+import argparse
+from dataclasses import dataclass
+from typing import List
+
+import asana
+import sys
+
+
+@dataclass
+class EnumOption:
+    name: str
+    color: str
+
+
+@dataclass
+class CustomField:
+    name: str
+    enum_options: List[EnumOption]
+
+
+CUSTOM_FIELDS = [
+    CustomField(
+        name="PR Status",
+        enum_options=[
+            EnumOption(name="Open", color="green"),
+            EnumOption(name="Merged", color="purple"),
+            EnumOption(name="Closed", color="red"),
+        ],
+    ),
+    CustomField(
+        name="Build",
+        enum_options=[
+            EnumOption(name="Success", color="green"),
+            EnumOption(name="Failure", color="red"),
+        ],
+    ),
+]
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="""A helper script to set up an SGTM tasks Asana project with the suggested custom fields
+		"""
+    )
+
+    parser.add_argument(
+        "-p",
+        "--personal_access_token",
+        help="""The Asana Personal Access Token that will be used to set up the Asana project. If you do not already have one,
+		follow the instructions at https://developers.asana.com/docs/personal-access-token
+		""",
```

Comment:
Should there be some documentation why this would be a different access token than SGTM will actually use?

Reply:
I'm not sure what you mean - can you elaborate?

Reply:
In the setup of the project itself you need to specify an access token for the user who will be creating the tasks from SGTM. This could be the same or a different access token than the one they'd be passing in here. For example - our SGTM user right now at Asana is a guest, but I (@gregeinfrank) created the projects

Reply:
How about
```
The PAT you pass in here can be the same/different from the PAT created in the "Create your credentials" section
```
?

Reply:
Sure -- and do we want to fall back to the PAT in the environment variables if not passed in?